MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 graft mbed_tools_ci_scripts
 global-exclude *.py[cod] __pycache__ *.so
+global-include *.spdx
```

Comment:
Imma re-provision this with cookiecutter, so this is not necessary.

Reply:
How do you want me to proceed?
Do you want to create a separate PR where we sync using cookie cutter?